Makefile

```diff
@@ -19,8 +22,22 @@ create-application-user:
 	CREATE_SECRET=secret-application.yml.tpl \
 	sh ./db-ops/create-db-user.sh || echo
 
+create-application-dev-user:
```

Comment:
This shouldn't use parameters passed in from `zero apply` like `randomSeed`, etc.
It's fine for the other ones since they only have to be run once, but you would need to need to run this make target every time you add a new dev to the team.

Reply:
There should also be logic to not attempt this operation for users that already exist.

Reply:
decided to use 1 user/pass for different user databases, then about the logic to check which user db has been created.

(from Bill) Sounds like we need to tweak the script in db-ops for this anyway, so maybe that script can check. Plus maybe we could just call it once with a list of all users. then inside the pod it loops through the users, checks for the db, creates any that don’t exist.

Reply:
if we create only one AWS user/pass secret for multiple databases... it's hard for zero apply Makefile to get a logic to know which databases inside RDS have been created.... as we don't suppose to access RDS and do show databases from laptop when zero apply.

Reply:
(fron Bill) Sounds like we need to tweak the script in db-ops for this anyway, so maybe that script can check. Plus maybe we could just call it once with a list of all users. then inside the pod it loops through the users, checks for the db, creates any that don’t exist

Reply:
We have to suppose use `zero apply` everytime after add new dev, unless we move it to templates/ side but  db-ops/ is not there.

Reply:
`db-ops` was moved into `templates/` in #163

Reply:
done the merge with templates/ one and put some enhancements for full review again.